### PR TITLE
docs(carbon): create contributing-with-github-desktop.md guide

### PR DESCRIPTION
**Pull request by dakahn**
Thought it might be helpful to create a guide walking through the setup and workflow of contributing to Carbon using the more user-friendly Github Desktop application 🏄

### CODE REVIEW

File: docs/guides/setup/contributing-with-github-desktop.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+9. Back in the Github Desktop application click the button labeled 'Clone a repository from the internet...'
+10. In the modal that pops up you can search your account for the fork you just made. Once you've selected the right repository click the blue button labeled 'Clone...' in the bottom right corner of the modal.
+11. The _next_ modal is a confirmation screen where you can double check the URL of your fork and select where on your computer you'd like to clone the repository. 
+12. Once the Github Application is done cloning your repository select that you intend 'To contribute to the parent project' and then click the button labeled 'Continue'. You are now ready to work.
+
+## Basic workflow
+
+1. First things first we need to create a "working" branch. At the top of the Github Desktop application click the drop down labeled 'Current branch'
+2. Click the button labeled 'New branch' and then name your branch. Typically this will be a general description of the work youre doing. Maybe 'updating-accessibility-docs' or 'adding-security-pictograms'. Then click 'Create branch' to continue
```
Comment (aledavila) [inline docs/guides/setup/contributing-with-github-desktop.md]: `you’re` Is just missing the apostrophe.

**Review by aledavila — APPROVED** [review]
Other than the one comment. This is great work. Super helpful for the non dev/non git people!

File: docs/guides/setup/contributing-with-github-desktop.md (hunk 1 above)
Comment (dakahn) [reply docs/guides/setup/contributing-with-github-desktop.md]: Honestly pleasantly surprised that was the only obvious grammatical error.

**Comment by dakahn** [comment]
Oh, Alex. Went ahead and removed the word 'just'

**Review by tw15egan — APPROVED** [review]
Nice addition! 🤘 ✅

**Review by joshblack — APPROVED** [review]

**Pull request merged**